packages/i18n/src/messages/errorMessages.ts

```diff
@@ -122,6 +122,12 @@ const errorMessages = defineMessages({
     id: "qNERoQ",
     description: "Message for when the download request failed",
   },
+  downloadingFileFailed: {
+    defaultMessage:
+      "There was a problem on our end. {fileName} failed to download. If you continue to receive this error, please get in touch with our support team",
```

Comment:
Missing a period at the end of the final sentence.

Reply:
Fixed in [fix typos](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10992/commits/03a84f9ea9c3dadcc062a72d6e6a27407af29a99)